afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/EventsBus.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.storage;
+
+import com.powsybl.afs.storage.events.AppStorageListener;
+import com.powsybl.afs.storage.events.NodeEvent;
+
+/**
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
+ */
+public interface EventsBus {
+    void pushEvent(NodeEvent event, String topic);
```

Comment:
I think we should clarify what the topic could be used for :
 * listeners are not associated to a topic, neither events when listeners are notified
 * I think in current implementations topics are not really used

Reply:
topic in the pushEvent function signature is the queue name in which message will be inserted, so when we want to send and Event we have to identify the topic name. that's will be more useful when we'll introduce other types of events..